library/script/__init__.py

```diff
@@ -7,3 +9,14 @@ def df_to_tempfile(df: pd.DataFrame) -> str:
     f = tempfile.NamedTemporaryFile(delete=False, suffix=".csv")
     df.to_csv(f, index=False)
     return f.name
+
+
+def get_json_content(url: str) -> str:
+    # use default headers to prevent "403 Client Error: Forbidden for url"
```

Comment:
Nice workaround to the 403 error. The user agent refers to the browser?

Reply:
Saw your comment in the PR about the user agent - doesn't seem super intuitive how to get the useragent

Reply:
unfortunately it isn't, and using the one produced by navigating to the site with my browser didn't work

the one that did work (and is in the code) is practically a random one people suggested using